_episodes/09-cmorization.md

```diff
@@ -123,6 +123,12 @@ run the CMORizer scripts:
 esmvaltool data format --config_file <path to config-user.yml>  <dataset-name>
 ```
 
+The options `--start` and `--end` can be added to command above to restrict the
+formatting of raw data to a time range. They will be ignored if a specific
+dataset does not support (i.e. because it is provided as a single file).
```

Comment:
Suggested change: dataset does not support -> dataset does not support this option (i.e because all the data is provided as a single file).